csrc/quantization/activation_kernels.cu

```diff
@@ -0,0 +1,119 @@
+#include <ATen/cuda/CUDAContext.h>
+#include <torch/all.h>
+#include <c10/cuda/CUDAGuard.h>
+
+#include <cmath>
+#include "../core/math.hpp"
+#include "../cuda_compat.h"
+#include "../dispatch_utils.h"
+
+using FP8_TYPE = c10::Float8_e4m3fn;
+C10_HOST_DEVICE constexpr auto FP8_E4M3_MAX =
+    std::numeric_limits<FP8_TYPE>::max();
+// using FP8_TYPE = c10::Float8_e4m3fnuz;
```

Comment:
delete this before landing if unneeded?

Alternatively this is the FP8 type we'd need to use for AMD GPUs so could add conditional compilation guards to use this instead of `Float8_e4m3fn` on RoCM

Reply:
Yeah we should probably add some RoCM guards.